index.js

```diff
@@ -149,10 +155,15 @@ SQLdown.prototype._get = function (key, options, cb) {
     if (!res.length) {
       return cb(new Error('NotFound'));
     }
+
     try {
       var value = JSON.parse(res[0].value);
       if (asBuffer) {
-        value = new Buffer(value);
+        if(value.type === 'Buffer'){
```

Comment:
yeah I meant just stringify the value into the buffer like you are which means you can get rid of the top half of this conditional